pkg/machines/manifest.json.in

```diff
@@ -9,5 +9,6 @@
         "path": "index.html",
         "order": 60
      }
-  }
+  },
+  "content-security-policy": "default-src 'self' 'unsafe-inline' 'unsafe-eval' data:"
```

Comment:
This would be the first upstream cockpit module with `unsafe-*`. For what part  is that needed? This should be justified in a comment. Can this be avoided somehow? Is that only necessary for the inline noVNC view, or already for the .vv downloads?